src/main/java/duke/storage/Storage.java

```diff
@@ -100,6 +100,9 @@ public ArrayList<Task> getTasks() {
      * @throws DukeException - Error of the filed not being found.
      */
     public ArrayList<Task> load() throws DukeException {
+        //@@author pzaiming-reused
+        //Reused from https://github.com/RyanQiu1
+        // with minor modifications
```

Comment:
I cannot comment on the method `Storage.updateStorage()` because it was not modified in this commit. However, I think the Javadoc of this method does not abide by coding convention. 

I think `This method will update the textfile based on the current commands inputed by the user.` should be changed to ` Updates the textfile based on the current commands inputed by the user` to align with convention.

Example of a Javadoc comment:

```Java
/**
 * Returns lateral location of the specified position.
 * If the position is unset, NaN is returned.
 *
 * @param x X coordinate of position.
 * @param y Y coordinate of position.
 * @param zone Zone of position.
 * @return Lateral location.
 * @throws IllegalArgumentException  If zone is <= 0.
 */
public double computeLocation(double x, double y, int zone)
        throws IllegalArgumentException {
    //...
}
```